### ISSUE TITLE
[Feature section]: VO does not read out image

### ISSUE THREAD

**Issue by oliviaflory** [carbon-design-system/carbon-for-ibm-dotcom#8131]
### Description

Voice over does not read the image in Feature section in the React wrapper.

https://user-images.githubusercontent.com/45692622/150820496-90db23e7-8005-415a-8bcb-78eeccebafea.mov

Expected:
VO does read the image in Web components.

_This is present in Production_

### Component(s) impacted

Feature section

### Browser

Chrome

### Carbon for IBM.com version

v1.15.0 rc 0

### Severity

Severity 3 = The problem is visible or noticeable to users but does not impede the usability or functionality. Affects minor functionality, has a workaround.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-react

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/next/web-components-react/iframe.html?id=components-feature-section--default&knob-Pictogram%20position:_PictogramCard=bottom&knob-Heading:_PictogramCard=Aerospace%20and%20defence&knob-Card%20style:_PictogramCard=none&knob-Add%20image:_StaticCard=true&knob-Eyebrow:_StaticCard=SPSS%20Statistics&knob-Heading:_StaticCard=Free%20trial&knob-Body%20copy:_StaticCard=Enjoy%20full%20SPSS%20Statistics%20capabilities%20including%20all%20add-ons.%20All%20trial%20registrants%20are%20restricted%20to%20one%20free%20trial%20per%20computer%20per%20user.&knob-Outlined%20card:_StaticCard=true&knob-Add%20tags:_StaticCard=true&knob-Add%20CTA:_StaticCard=true&knob-Alignment=left&knob-Media%20type=image&knob-Image%20alt%20text=Image%20alt%20text&knob-Heading%20(heading):=Aliquam%20condimentum&knob-CTA%201%20type%20(cta-type):=local&knob-CTA%201%20copy%20(cta-copy):=Learn%20more&knob-CTA%201%20href%20(cta-href):=https://www.ibm.com&knob-CTA%202%20type%20(cta-type):=external&knob-CTA%202%20copy%20(cta-copy):=Microservices%20and%20containers&knob-CTA%202%20href%20(cta-href):=https://www.ibm.com&knob-CTA%20Block%20border_CTASection=true

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues
